content/en/docs/reference/node/cri-api-development-policies.md

```diff
@@ -0,0 +1,107 @@
+---
+content_type: "reference"
+title: Kubernetes feature development and Container runtimes
+weight: 10
+---
+
+The mechanics of a feature development that requires new CRI APIs is covered in
+documentation on CRI API [feature-development](https://github.com/kubernetes/cri-api?tab=readme-ov-file#feature-development).
+This article declares policies for developing new Kubernetes features
+that require CRI API changes. The goal of these policies is to ensure great user
+experience for people trying the new feature early, adopting it when it is
+enabled by default, and relying on it as a GA functionality.
+
+## Supported container runtimes
```

Comment:
We cannot risk implying that runtimes not on this list are unsupported. I think we must change this text.

I think this is a list of runtimes that this project uses for its own release testing. It's not related to conformance, for example.